muonShieldOptimization/makeMuonDIS.py

```diff
@@ -1,4 +1,8 @@
 import ROOT,time,os,sys
+
+ROOT.gROOT.LoadMacro("$VMCWORKDIR/gconfig/basiclibs.C")
```

Comment:
Can we make this a header and load it as such?
I'd really like to limit our use to macros as much as possible.

Reply:
For now (and to minimize errors on my changes), I have followed the procedure already in place for makeCascade.py 

But I believe it should work as you suggested

Reply:
Ok, then it's fine for now.